docs/sources/enterprise/reporting.md

```diff
@@ -39,10 +39,45 @@ Currently only Organization Admins can create reports.
 1. Select the layout option for generated report: **Portrait** or **Landscape**.  
 1. Enter scheduling information. Options vary depending on the frequency you select.
 1. **Save** the report.
+1. **Send test mail** after saving the report to verify that the report looks like you expect it to.
 
 {{< docs-imagebox img="/img/docs/enterprise/reports_create_new.png" max-width="500px" class="docs-image--no-shadow" >}}
 
-## Send test mail
+### Scheduling
+
+Scheduled reports can be sent on a weekly, daily, or hourly basis. You may also disable scheduling for when you either want to pause a report or send it via the API.
+
+All scheduling is approximative and indicates when the reporting service will start rendering the dashboard.
```

Comment:
```suggestion
All scheduling indicates when the reporting service will start rendering the dashboard. It can take a few minutes to render a dashboard with a lot of panels.
```
It feels weird to start by `All scheduling is approximative`. It's not very positive about the feature.

Reply:
🤔 I kind of prefer my variant here. For two reasons. One is that it's important to note that the _start_ of rendering might not be on the dot, it may shift back and forth a little based on a number of reasons (mostly forwards). The other thing is that one of the things we said we wanted to improve was setting clear expectations for the feature, and I think it'll lead to less frustration when the documentation acknowledges tradeoffs straight away.

Reply:
I prefer @AgnesToulet 's suggested edit.

Reply:
I'll merge that then 🤷 But I still don't understand why, so I _will_ most likely do the same mistake in my next PR. I've learnt nothing here, and my response was specifically spelling out my reasons for my choices to open up for learning (even if it's just "it just feels better").

Reply:
Just talked with @xlson and explained my concerns with the difference and he pointed out that my concern about the approximative start position. He'll open an issue with more details and we'll fix it in the backend instead of documenting it.